eng/pipelines/global-build.yml

```diff
@@ -92,4 +92,17 @@ jobs:
     jobParameters:
       testGroup: innerloop
       nameSuffix: Mono_Libraries
-      buildArgs: -subset mono+libs /p:RuntimeFlavor=Mono
\ No newline at end of file
+      buildArgs: -subset mono+libs /p:RuntimeFlavor=Mono
+
+#
+# SourceBuild Build
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Linux_x64
+    jobParameters:
+      nameSuffix: SourceBuild
+      buildArgs: /p:DotNetBuildFromSource=true
```

Comment:
This will also build mono, do we care about mono?

Should we instead do, `-subset clr+libs+installer /p:DotNetBuildFromSource=true`

Reply:
> do we care about mono?

I was assuming that Mono indeed is a target for sourcebuild? cc @dseefeld @marek-safar

Reply:
There isnt any code specific to sourcebuild in mono, so we should be good by building only for coreclr runtime

Reply:
Don't want to make things complicated but it really depends on your definition of sourcebuild

Reply:
> There isnt any code specific to sourcebuild in mono, so we should be good by building only for coreclr runtime

Btw, if mono is green with `sourcebuild=true` it doesn't hurt to build it. I was just asking if it was relevant, but it doesn't hurt to have it, it only adds like 10 mins to the build.

Reply:
> Don't want to make things complicated but it really depends on your definition of sourcebuild

cc @dseefeld